addons/event/models/event_config_settings.py

```diff
@@ -31,7 +31,9 @@ class event_config_settings(osv.TransientModel):
             (1, 'Schedule emails to attendees and subscribers')
             ], "Email Scheduling",
             help='You will be able to configure emails, and to schedule them to be automatically sent to the attendees on subscription and/or attendance',
-            implied_group='event.group_email_scheduling'),            
+            implied_group='event.group_email_scheduling'),
+        'module_event_barcode': fields.boolean("Scan badges to confirm attendances",
```

Comment:
@mart-e  , i have already put the selection but we can not apply widget upgrade_boolean(means two widget radio and upgrade_boolean) on Selection and @bst-odoo  said inn pad put enterprise upgrade button on that so i make it boolean :  https://drive.google.com/a/openerp.com/file/d/0BxqHe5rtIxY0cExNeXNJd19adjA/view?usp=drivesdk  thanks